lib/resources/r4/communication.yaml

```diff
@@ -177,7 +157,7 @@ fields:
   type: BackboneElement
   description: The text of the communication
   url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.payload
-  note: Content must be supplied as a content[Attachment] object with the contentType and data defined.  For contentType, application/xhtml+xml or text/plain are allowable. data should be base64 encoded.
+  note: The content must be supplied as a contentAttachment with contentType and data defined.  For contentType, application/xhtml+xml or text/plain are supported, and data should be base64 encoded.
```

Comment:
nit: double space in front of 'For'

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/720c35bf50e9cb8dd953ad8a2bce0b6a3c29f74a#diff-f08ae1e4c063b9a36a22adaa43fecf21L160-R160